fuzzers/libfuzzer_libpng_cmin/src/bin/libafl_cc.rs

```diff
@@ -25,6 +25,7 @@ pub fn main() {
             .expect("Failed to parse the command line")
             .link_staticlib(&dir, "libfuzzer_libpng")
             .add_arg("-fsanitize-coverage=trace-pc-guard")
+            .add_arg("-lstdc++")
```

Comment:
:clown_face:

Reply:
This is dangerous. Some platforms will require `-lc++` instead of `-lstdc++`.

Perhaps you want to do something similar to what https://crates.io/crates/link-cplusplus does?

Reply:
That's a fair point -- this is for an example fuzzer, so it shouldn't be too great an issue, but I'll make the update nonetheless. Thanks!

Reply:
@s1341 I changed this by removing the link arg and forcing the compiler to C++. link-cplusplus does not work as libfuzzer_libpng is a static lib.